addons/missileguidance/functions/fnc_keybind_add.sqf

```diff
@@ -0,0 +1,23 @@
+#include "..\script_component.hpp"
+/*
+ * Author: PabstMirror
+ * Adds keybind handlers
+ *
+ * Arguments:
+ * 0: Type of keybind <STRING>
+ * 1: Down handler code <CODE>
+ * 2: Up handler code <CODE>
+ *
+ * Returns:
+ * None
+ *
+ * Example:
+ * ["left", {hint "Down"}, {hint "Up"}] call ace_missileguidance_fnc_keybind_add
+ *
+ * Public: No (maybe)
```

Comment:
should be a soft "no" imo. we shouldn't explicitly make this an API, but i think encouraging non-breaking changes is nice. missileguidance as a whole is shitty api wise

Reply:
```suggestion
 * Public: No
```